scripts/fix_format_pico_conll2003.py

```diff
@@ -0,0 +1,42 @@
+"""
+
+The processed PICO files for EBMNLP (see amandalynne's project)
+are formatted with 2 columns:  Token & Tag
+
+** CONLL2003 reader requires 4 columns.
+
+Also, the PICO data is separate files for each paper instead of 1 long file.
+
+** Concats those and adds --DOCSTART-- to mark the paaper IDS
+
+Also, PICO data is not sentence split, but BERT truncates if sequence is
+longer than 250 tokens.  This causes bugs.
+
+** Don't do anything fancy, just split on every '.' that isn't part of a PICO element.
+
+"""
+
+import os
+
+old_dirname = 'ebmnlp_amandalynne/'
```

Comment:
nit: replace hardcoded paths with args. I found `plac` to be very easy to use, e.g. https://github.com/allenai/sci-bert/blob/master/scripts/s2conll_to_conll2003.py

Reply:
done